modules/kubernetes-addons/karpenter/data.tf

```diff
@@ -1,20 +1,16 @@
 data "aws_arn" "queue" {
-  count = var.enable_spot_termination_handling ? 1 : 0
+  count = var.sqs_queue_arn != "" ? 1 : 0
```

Comment:
we should leave these as is - comparison checking is problematic. For example, if someone sets this value as null, things will fail

Reply:
Yes, you're right. In fact I don't really remember changing this count, but I anyway I reverted it.